package.json

```diff
@@ -16,8 +16,8 @@
     "react-dom": "^16.13.0",
     "react-router-dom": "^5.1.2",
     "redux-zero": "^5.1.3",
-    "virtru-sdk-current": "npm:virtru-sdk@^2.0.1",
-    "virtru-sdk-lts": "npm:virtru-sdk@^1.6.16"
+    "virtru-sdk-current": "npm:virtru-sdk@^2.1.0-beta.19",
```

Comment:
build failing: "Please update your lock file with `npm install` before continuing."

Reply:
Woah... I've never seen this before. What does prepending with `npm:` do?